### ISSUE TITLE
PS3 occasionally freezes when mounting PS2 ISOs

### ISSUE THREAD

**Issue by Lortess** [aldostools/webMAN-MOD#1135]
Hi aldostoools, i have an issue with webman that whenever i mount a PS2 ISO sometimes, i’d say 1 in 4 times, the PS3 freezes. It always goes like this: the PS2 manual fan always starts correctly and the notification info appears on screen, but then the screen freezes, controller doesn’t work, no beeps or blinking Led and i have to manually shutdown the power button, after that it does the 3 beeps and then shuts, when i boot it back up i have the usual incorrect shutdown message. It happens only with PS2 ISOs and it can happen with all of them, i already tested this. The freeze also doesn’t only happen when i mount the first ISO after booting the PS3, it can happen at the second, third, or fourth ISO that i mount.

To clarify, i have a PS3 model CECH-2500B, I am on CFW Evilnat 4.91.2 BETA 12 PEX and webman mod full 1.47.47g, but i have this issue since Evilnat 4.90 CEX and webman mod full 1.47.45. I already tried changing memory usage both upping or diminishing it but to no avail.
I don’t have other plugins besides the ftp one that comes bundled with the full version of webman, and i have only the latest version of multiman as homebrew besides webman mod.

Sorry for the long post but i feel like i have tried everything and the issue still persist😑.

**Comment by aldostools** [comment]
Hello @Lortess 
I suspect that the freeze could be related to the fan speed set by either CFW Evilnat or webMAN MOD.

To confirm if it's an issue caused by webMAN MOD, delete `/dev_hdd0/boot_plugins.txt`, reboot and mount the PS2 game with multiMAN (mmCM) or IRISMAN. If the issue doesn't happen, that confirms that webMAN MOD is producing the freeze.

Also try resetting Cobra configuration to the default settings, deleting `/dev_hdd0/vm/cobra_cfg.bin` and rebooting.

Other possible tests: Reinstall webMAN MOD and do the following (try one at a time):
- Try resetting webMAN MOD configuration to the default settings, deleting `/dev_hdd0/tmp/wm_config.bin` and rebooting.
- Try setting fan speed for PS2 to 40% or 50% in /setup.ps3 (don't use lower than 30% or higher than 80%)
- Try installing a very old version like 1.47.34 to check if it's related to changes in recent versions. If it works, you can install higher versions until you find which version is causing the issue.

**Comment by Lortess** [comment]
Hi aldo, thank you for your time.
I always used webman mod dynamic fan control set 31 to 56 and PS2 manual fan set to 36, i just bumped the PS2 fan to 46 but i still get the occasional freeze. This morning i tried completely uninstalling all webman files in /devhdd0 and doing a fresh install but i still have the issue.

The things i haven’t tried yet are mounting ISOs through multiman or downgrading webman to an older version, i’ll let you know how it went.

**Comment by Lortess** [comment]
Sorry, never used github, i think i clicked something wrong, thank you for your time by the way.
I always used used webman dynamic fan set 31 to 56 and PS2 fan set to 36, i just tried upping it to 46 but the issue is still there. I also tried this morning to uninstall webman and perform a fresh install but the result is the same.

I have yet to try mounting on multiman or downgrading to an older version of webman, i’ll let you know how it went.

**Comment by Lortess** [comment]
Ok, it went like this: i deleted boot_plugins.txt and also cobra_cfg.bin and rebooted, then i opened multiman and tried mounting an ISO from there, pressed x on the icon and black screen, XMB restarted but the game was not mounted and webman games disappeared. I tried with multiple ISOs from PS1 and PSP but multiman doesn’t mount anything at all.

How do i access webman now, do i have to reinstall it? If so what are the exact files i have to delete before reinstalling it?

By the way multiman is version 4.91 MOD and i still have to try an older version of webman.

**Comment by Lortess** [comment]
I manually uninstalled webman and tried a fresh install of 1.47.34 without touching anything in setup, can confirm the freeze still happens. I will now try using IRISMAN and Managunz and see how it goes

**Comment by aldostools** [comment]
For multiMAN, it should be in mmCM mode and Cobra enabled.

Aparently it is not an issue with webMAN MOD or recent builds. It could be a hardware issue with your console.

**Comment by Lortess** [comment]
Multiman continues to black screen itself after a few seconds and restart XMB even after repeatedly deleting and reinstalling it, i managed to mount an ISO only once with it. After one last reinstall the repeated black screen stopped but i still can’t mount ISOs with it, when i try to mount one mm closes, XMB restarts but there is no disc icon. When i boot it the title says mmCM and Cobra is enabled. 

IRISMAN seems to work.

**Comment by aldostools** [comment]
The black screen could be due multiMAN MOD scans the exFAT + NTFS drives each time it's started.

It's weird that multiMAN is not working. webMAN MOD and IRISMAN use the same Cobra opcodes to mount PS2ISO.

**Comment by Lortess** [comment]
It seems that the problem with multiman crashing and not mounting ISOs was indeed my USB drive plugged in, after unplugging it it now works as intended; for some reason IRISMAN wasn’t bothered by the pendrive to function properly.

So, as of now: Multiman and IRISMAN both work as they should while webman mod continues to have its occasional freezes. I’ll try testing some more.

**Comment by aldostools** [comment]
Great! At least now you know that you can play PS2ISO with these applications.

**Comment by Lortess** [comment]
Indeed, but it’s a pity that i can’t reliably mount PS2 ISOs with webman mod because i prefer doing everything in XMB without loading homebrew, it’s more streamlined. I guess nothing’s perfect.

**Comment by aldostools** [comment]
You can test if PS2 Classics mounted through webMAN MOD are working.

You can try changing the ps2_netemu in webMAN Setup > Update webMAN MOD > Custom Firmware MODs menu.
Although I don't think the issue it related to the emulator, since the other backup managers work fine.

The major difference between these apps and webMAN MOD, is the automatic change of fan settings to manual when a PS2ISO is mounted and the auto-config installation that can be disabled in /setup.ps3

**Comment by Lortess** [comment]
Thanks for the feedback, i was thinking the same thing in trying PS2 Classics although i only used ISOs so far, i also use my own config files and already disabled the auto-config function so the only thing webMAN does is the manual fan speed. The strange thing is that the freeze is completely random, sometimes it works even with 15 consecutive mounts and sometimes it freezes on the first, second or third mount. I retried with different memory usage settings but the result is the same.

**Comment by aldostools** [comment]
The memory usage configuration is mainly used during the file listing, game scanning and FTP downloads.

Other tests that you perform are: disabling fan control, mounting the PS2ISO from PC, uncheck all categories and only leave PS2 enabled. Disable FPS counter if you have it enabled. Remove other plugins from boot_plugins.txt if you're using others.

**Comment by Lortess** [comment]
I don't use FPS counter and other plugins so those are not an issue but i never tried disabling fan control, not sure what mounting from PC means. By unchecking categories you mean the files that webMAN scans for content?

**Comment by aldostools** [comment]
By unchecking categories I mean select to scan only PS2 games in /setup.ps3

For mounting from PC, smartphone or tablet, open Chrome or Safari, enter the PS3 IP address in the address bar, select GAMES button and click on the game.

About the other plugins, I only mentioned them in case you were using other background applications that can cause issues in webMAN MOD's opetation.

**Comment by Lortess** [comment]
Ah, didn't know i could mount from outside the PS3, i'll also try scanning only PS2 games and disabling fan control. 

Thank you for your time, if anything changes i'll let you know 💪.

**Comment by Lortess** [comment]
Tested enabling scan only for PS2 games and disabling the fan control, the freeze can still happen.

At this point it seems to me that, at least on my specific setup, it’s a webMAN MOD issue.

**Comment by aldostools** [comment]
I understand. However I haven't received reports from other users experiencing your issue and since it also happens in a very old version it would have been reported in all these years, if it would be a webMAN MOD issue.

I would say that it's likely something with your setup or hardware. Maybe an SSD issue, with the ISOs or configuration.

**Comment by Lortess** [comment]
Yeah, i’ll try to delete and reinstall all PS2 ISOs to see if there’s any corruption, the HDD is still the PS3 original since it has 320 gb and it’s a barely used PS3, maybe someday i’ll try to change it.

**Comment by Joonie86** [comment]
No further feedback received
